### PR TITLE
Update tutorial with new tags

### PR DESCRIPTION
**Pull request by jasro23**
Signed-off-by: Jason Roche <jroche@nvidia.com>

### CODE REVIEW

**Comment by jasro23** [comment]
Update the tutorials with
1. Volume attribute under Prosody.
2. Added examples for sub
3. Added exam[;es for Emphasis

**Comment by blisc** [comment]
Can you do the following

- Where it says `Riva TTS supports two SSML tags:` can you make this accurate?
- `Customizing pronunciation with the phenome tag` is misspelt

Does github display the audio output? I don't see it in the PR

**Comment by blisc** [comment]
can you target release/22.10 instead of stable?

**Comment by jasro23** [comment]
> Can you do the following
> 
>     * Where it says `Riva TTS supports two SSML tags:` can you make this accurate?
> 
>     * `Customizing pronunciation with the phenome tag` is misspelt
> 
> 
> Does github display the audio output? I don't see it in the PR

Fixed the typos

File: tts-python-advanced-customization-with-ssml.ipynb [hunk 1]
```diff
@@ -317,6 +343,104 @@
...
+    "\n",
+    "sample_rate_hz = 44100\n",
+    "# Request to Riva TTS to synthesize audio\n",
+    "resp = riva_tts.synthesize(\n",
+    "    text=ssml_text,\n",
+    "    voice_name=\"English-US.Female-1\",\n",
+    "    language_code=\"en-US\",\n",
+    "    encoding=riva.client.AudioEncoding.LINEAR_PCM,\n",
+    "    sample_rate_hz=sample_rate_hz,\n",
+    ")\n",
```
Comment (blisc) [inline tts-python-advanced-customization-with-ssml.ipynb:385]: Let's clean this notebook up. We don't have to re-define this everytime. Let's just define this once at the beginning of the notebook and change the text field. `resp.text = ssml_text`

Can we do the pip install of riva-client in the notebook itself?

**Comment by blisc** [comment]
Please add sample audios. Just upload the wav files to this repo and link them in the notebook

File: tts-python-advanced-customization-with-ssml.ipynb (hunk 1 above)
Comment (jasro23) [reply tts-python-advanced-customization-with-ssml.ipynb:385]: I can set up a dictionary and pass it as arguments.
>Can we do the pip install of riva-client in the notebook itself?
I need to try this out

File: tts-python-advanced-customization-with-ssml.ipynb [hunk 2]
```diff
@@ -167,28 +200,32 @@
...
     "audio_samples = np.frombuffer(resp.audio, dtype=np.int16)\n",
     "ipd.display(ipd.Audio(audio_samples, rate=sample_rate_hz))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Expected results if you run the notebook:\n",
+    "`<prosody pitch='2.5'>Today is a sunny day</prosody>. <prosody rate='high' volume='+1dB'>But it might rain tomorrow.</prosody>` \\\n",
```
Comment (blisc) [inline tts-python-advanced-customization-with-ssml.ipynb:225]: Can you remove change `\\\n` to `\n`?

File: tts-python-advanced-customization-with-ssml.ipynb [hunk 3]
```diff
@@ -235,7 +267,37 @@
...
+    "`<prosody pitch=\"-0.5\">And now, I'm speaking a bit lower</prosody>` \\\n",
+    "<audio controls src=\"https://raw.githubusercontent.com/nvidia-riva/tutorials/stable/audio_samples/tts_samples/ssml_sample_6.wav\" type=\"audio/ogg\"></audio> \n",
+    "\n",
+    "`S S M L supports <prosody pitch=\"-1\">nested tags. So I can speak <prosody rate=\"150%\">faster</prosody>, <prosody rate=\"75%\">or slower</prosody>, as desired.</prosody>` \\\n",
+    "<audio controls src=\"https://raw.githubusercontent.com/nvidia-riva/tutorials/stable/audio_samples/tts_samples/ssml_sample_7.wav\" type=\"audio/ogg\"></audio> \n",
+    "\n",
+    "`<prosody volume='x-soft'>I'm speaking softly.</prosody><prosody volume='x-loud'> And now, This is loud.</prosody>` \\\n",
+    "<audio controls src=\"https://raw.githubusercontent.com/nvidia-riva/tutorials/stable/audio_samples/tts_samples/ssml_sample_8.wav\" type=\"audio/ogg\"></audio>\n"
+   ]
+  },
```
Comment (blisc) [inline tts-python-advanced-customization-with-ssml.ipynb:295]: Can you remove change \\\n to \n?

File: tts-python-advanced-customization-with-ssml.ipynb [hunk 4]
```diff
@@ -167,28 +200,32 @@
...
     "# Playing the generated audio from Riva TTS request\n",
     "audio_samples = np.frombuffer(resp.audio, dtype=np.int16)\n",
     "ipd.display(ipd.Audio(audio_samples, rate=sample_rate_hz))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Expected results if you run the notebook:\n",
```
Comment (blisc) [inline tts-python-advanced-customization-with-ssml.ipynb:224]: Let's add a note here. If you don't see audio, please go to the docs and link the riva docs, or open it in github dev

File: tts-python-advanced-customization-with-ssml.ipynb (hunk 3 above)
Comment (jasro23) [reply tts-python-advanced-customization-with-ssml.ipynb]: fixed

File: tts-python-advanced-customization-with-ssml.ipynb (hunk 4 above)
Comment (jasro23) [reply tts-python-advanced-customization-with-ssml.ipynb:224]: added a note for the first sample.

**Review by blisc — APPROVED** [review]

**Pull request merged**